optimum/exporters/openvino/model_configs.py

```diff
@@ -3236,3 +3239,168 @@ def get_model_for_behavior(self, model, behavior: Union[str, VLMConfigBehavior])
+            int_dtype=int_dtype,
+            float_dtype=float_dtype,
+            use_past=use_past,
+            use_past_in_inputs=use_past_in_inputs,
+            behavior=behavior,
+            preprocessors=preprocessors,
+            is_postnet_and_vocoder=False,
+            legacy=legacy,
+        )
+        self.is_postnet = is_postnet
+        self.is_vocoder = is_vocoder
+
+    def patch_model_for_export(
+        self, model: Union["PreTrainedModel", "TFPreTrainedModel"], model_kwargs: Optional[Dict[str, Any]] = None
+    ) -> "ModelPatcher":
+        return OVSpeechT5ModelPatcher(self, model, model_kwargs=model_kwargs)
+
+    def add_past_key_values(self, inputs_or_outputs: Dict[str, Dict[int, str]], direction: str):
+        if direction not in ["inputs", "outputs"]:
+            raise ValueError(f'direction must either be "inputs" or "outputs", but {direction} was given')
+
+        if direction == "inputs":
+            decoder_sequence_name = "past_decoder_sequence_length"
+            name = "past_key_values"
+        else:
+            decoder_sequence_name = "past_decoder_sequence_length + 1"
+            name = "present"
+
+        for i in range(self._normalized_config.decoder_num_layers):
+            inputs_or_outputs[f"{name}.{i}.decoder.key"] = {0: "total_batch_size", 2: decoder_sequence_name}
+            inputs_or_outputs[f"{name}.{i}.decoder.value"] = {0: "total_batch_size", 2: decoder_sequence_name}
+            inputs_or_outputs[f"{name}.{i}.encoder.key"] = {0: "total_batch_size", 2: "encoder_sequence_length_out"}
+            inputs_or_outputs[f"{name}.{i}.encoder.value"] = {0: "total_batch_size", 2: "encoder_sequence_length_out"}
+
+    @property
+    def inputs(self) -> Dict[str, Dict[int, str]]:
+        common_inputs = {}
+        # Batched inference is not supported in Transformers.
+        if self._behavior is ConfigBehavior.ENCODER:
+            common_inputs["input_ids"] = {1: "encoder_sequence_length"}
+        elif self._behavior is ConfigBehavior.DECODER:
+            common_inputs["inputs_embeds"] = {0: "batch_size", 1: "decoder_sequence_length"}
+            common_inputs["speaker_embeddings"] = {}  # No dynamic shape here.
+            common_inputs["encoder_hidden_states"] = {0: "batch_size", 1: "encoder_sequence_length"}
+            common_inputs["encoder_attention_mask"] = {0: "batch_size", 1: "encoder_sequence_length"}
+            if self.variant == "with-past" and self.use_past_in_inputs:
+                self.add_past_key_values(common_inputs, direction="inputs")
+        elif self.is_postnet:
+            common_inputs["raw_spectrogram"] = {
+                0: "n_spectrums",
+                1: "batch_size",
+                2: "reduction_factor",
+                3: "num_mel_bins",
+            }
+        elif self.is_vocoder:
+            common_inputs["spectrogram"] = {0: "batch_size", 1: "n_spectrums", 2: "num_mel_bins"}
+        elif self.is_postnet_and_vocoder:
```

Comment:
I do not see this parameter initialization in config, is it expected?

Reply:
Removed